08_files_lab/01.py

```diff
@@ -1,6 +1,17 @@
-"""
-Write a program that takes 2 file names
-and appends the second file's contents to
-the end of the first one
-"""
-
+"""
+      _      _      _
+   __(.)< __(.)> __(.)=
+   \___)  \___)  \___)   
+
+Python course - Verint
+Week 2
+
+"""
+
+import sys
+
+with open(sys.argv[1], "r") as fin:
+    with open(sys.argv[2], "a") as fout:
+        fout.write("\n")                

```

Comment:
In order to have seperation between the files,
Without using the newline the output will be like:

![image](https://cloud.githubusercontent.com/assets/20268575/16761221/e5f57b42-4827-11e6-836a-c63b4ddbc055.png)